### DISCUSSION TITLE
NEW VISIONS For Quantum Computing and Music [Berlin, DE] (2024-09-06)

### DISCUSSION THREAD

**Discussion by omarcostahamido** [Quantumland-art/community#69] [Events]
**Where:** Pfefferwerk Stage, New Visions for music and sound, Berlin, DE
**When:** September 6, 2024
**URL:** https://newvisions.berlin/sessions/new-visions-for-quantum-computing-and-music/
![](https://newvisions.berlin/wp-content/uploads/2024/05/Home-1.png)

New Visions for Music and Sound Festival
Keynote _NEW VISIONS For Quantum Computing and Music_
by Spencer Topel

This groundbreaking keynote from Spencer Topel explores the transformative impact quantum computing will have on the music ecosystem. Despite its vast potential, few researchers, artists, or music industry professionals have delved into how quantum computing could impact music.
We are thrilled to have Spencer Topel from [Moth Quantum](https://mothquantum.com/) join us to share his visionary insights and the pioneering work of Moth Quantum, the first company to enable music production using qubits. Topel will illuminate the audience on the incredible potential of quantum computing, unveiling the various applications it holds for music. 
From unprecedented sound synthesis to complex data analysis, quantum computing offers new horizons for creativity and efficiency in music production.
Attendees will gain a deep understanding of how quantum computing can revolutionize music production, offering unprecedented possibilities for creativity and innovation. Don’t miss this chance to explore the intersection of quantum technology and music with one of the pioneers and leading experts in the field.

**Comment by Itaborala** [comment]
Great that this is here! I was about to write about it

**Comment by omarcostahamido** [comment]
😉
